src/main/scala/com/codacy/client/bitbucket/v2/service/AuthorizationServices.scala

```diff
@@ -0,0 +1,28 @@
+package com.codacy.client.bitbucket.v2.service
+
+import com.codacy.client.bitbucket.client.{BitbucketClient, Request, RequestResponse}
+import com.codacy.client.bitbucket.v2.RefreshToken
+import play.api.libs.json._
+
+class AuthorizationServices(client: BitbucketClient) {
+
+  sealed trait RefreshCredentials
+
+  case class OAuth2RefreshCredentials(grant_type: String, refresh_token: String) extends RefreshCredentials
```

Comment:
Please move to its own file near the other dtos/case classes.

`OAuth2RefreshCredentials` could be the exact name in the grant_type field. Also in the case class the grant_type should be static since it is one for that type of case class.